website/static/css/pages/contributor-page.css

```diff
@@ -35,4 +42,361 @@
 .copy-button {
     margin-right: 0;
     border-right: none;
-}
\ No newline at end of file
+}
+
+.contrib-placeholder {
+    background: #EFEFEF;
+    width: 100%;
+    height: 40px;
+}
+
+.sortable-bars {
+    display: inline-block;
+    position: absolute;
+    top: 50%;
+    transform: translate(0, -50%);
+    -webkit-transform: translate(0, -50%);
+    -ms-transform: translate(0, -50%);
+    color: #ccc;
+    cursor: pointer;
+}
+
+#noContributors {
+    display: none;
+}
+
+#noAdminContribs {
+    display: none;
+}
+
+select.input-sm {
+    white-space: nowrap;
+    min-width: 111px;
+}
+
+#manageContributorsTableth:first-child(:not(.sortable)),
+#adminContributorsTable th:first-child(:not(.sortable)) {
+    width: 40px;
+}
+
+#manageContributorsTable th:first-child.sortable,
+#adminContributorsTable th:first-child.sortable {
+    width: 87px;
+    text-indent: 20px;
+}
+
+#manageContributorsTable th:nth-child(4),
+#adminContributorsTable th:nth-child(4) {
+    width: 109px;
```

Comment:
This is a bit fragile. If we move columns around this will break while if column headers had classes on them and their classes had width definitions they wouldn't. I admire your overall selector use but this might be going a bit far. I recommend adding header specific classes like ```th-bibl-contrib```.